### PR TITLE
Ser2net multiconnections

### PR DESCRIPTION
**Pull request by montjoie**

### CODE REVIEW

File: lava-slave/Dockerfile [hunk 1]
```diff
@@ -22,6 +22,13 @@ RUN rm /etc/apt/sources.list.d/testing.list
 RUN apt-get -y install ser2net
 COPY ser2net.conf /etc
 
+# ser2net > 3.2 is only availlable from sid
+RUN echo "deb http://deb.debian.org/debian/ sid main" >> /etc/apt/sources.list.d/sid.list
+RUN apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y install ser2net
+RUN rm /etc/apt/sources.list.d/sid.list
+RUN apt-get update
+
```
Comment (khilman) [inline lava-slave/Dockerfile:31]: Adding 'sid' here is also going to affect any of the packages that get installed after this though, right?  Like the lava-cli deps?

File: README.md [hunk 2]
```diff
@@ -243,6 +243,7 @@ slaves:
     remote_user: 		the user used for connecting to the master
     remote_proto:		http(default) or https
     default_slave:		Does this slave is the default slave where to add boards (default: lab-slave-0)
+    export_ser2net:		Does ser2net ports need to be availlable on host
```
Comment (khilman) [inline README.md]: typo: available

Comment (khilman) [reply README.md]: prof d'anglais: Do ser2net ports need...

File: lava-slave/Dockerfile (hunk 1 above)
Comment (montjoie) [reply lava-slave/Dockerfile:31]: I remove it and apt-get update just after using it

**Review by khilman — APPROVED** [review]

**Pull request merged**